### PR TITLE
[Feature] Support Video Swin Transfomer

### PR DESCRIPTION
**Pull request by Dai-Wenxun**
## Modification

Support Video Swin Transfomer.

### CODE REVIEW

File: configs/_base_/models/swin/swin_large.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+_base_ = 'swin_tiny.py'
+model = dict(
```
Comment (hukkai) [inline configs/_base_/models/swin/swin_large.py:2]: Is it a good idea to add multiple configs files to `_base_/models`? They can be added to each  config file at `recognition/swin` actually

File: configs/recognition/swin/swin-large_p244-w877-in22k-pre_16xb8-amp-32x2x1-30e_kinetics700-rgb.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+model = dict(cls_head=dict(num_classes=700))
+
+dataset_type = 'VideoDataset'
+data_root = 'data/kinetics700/videos_train'
+data_root_val = 'data/kinetics700/videos_val'
+ann_file_train = 'data/kinetics700/kinetics700_train_list_videos.txt'
+ann_file_val = 'data/kinetics700/kinetics700_val_list_videos.txt'
+ann_file_test = 'data/kinetics700/kinetics700_val_list_videos.txt'
+
+file_client_args = dict(
```
Comment (hukkai) [inline configs/recognition/swin/swin-large_p244-w877-in22k-pre_16xb8-amp-32x2x1-30e_kinetics700-rgb.py:14]: remove file_client_args

File: configs/recognition/swin/swin-large_p244-w877-in22k-pre_8xb8-amp-32x2x1-30e_kinetics400-rgb.py [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+# dataset settings
+dataset_type = 'VideoDataset'
+data_root = 'data/kinetics400/videos_train'
+data_root_val = 'data/kinetics400/videos_val'
+ann_file_train = 'data/kinetics400/kinetics400_train_list_videos.txt'
+ann_file_val = 'data/kinetics400/kinetics400_val_list_videos.txt'
+ann_file_test = 'data/kinetics400/kinetics400_val_list_videos.txt'
+
+# file_client_args = dict(
```
Comment (hukkai) [inline configs/recognition/swin/swin-large_p244-w877-in22k-pre_8xb8-amp-32x2x1-30e_kinetics400-rgb.py:19]: remove file_client_args comments.

File: configs/recognition/swin/swin-small_p244-w877-in1k-pre_8xb8-amp-32x2x1-30e_kinetics400-rgb.py [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+# dataset settings
+dataset_type = 'VideoDataset'
+data_root = 'data/kinetics400/videos_train'
+data_root_val = 'data/kinetics400/videos_val'
+ann_file_train = 'data/kinetics400/kinetics400_train_list_videos.txt'
+ann_file_val = 'data/kinetics400/kinetics400_val_list_videos.txt'
+ann_file_test = 'data/kinetics400/kinetics400_val_list_videos.txt'
+
+# file_client_args = dict(
```
Comment (hukkai) [inline configs/recognition/swin/swin-small_p244-w877-in1k-pre_8xb8-amp-32x2x1-30e_kinetics400-rgb.py:19]: remove file_client_args comments

**Review by hukkai — CHANGES_REQUESTED** [review]
Please include corresponding UTs for the add codes.

File: configs/recognition/swin/swin-large_p244-w877-in22k-pre_16xb8-amp-32x2x1-30e_kinetics700-rgb.py (hunk 2 above)
Comment (Dai-Wenxun) [reply configs/recognition/swin/swin-large_p244-w877-in22k-pre_16xb8-amp-32x2x1-30e_kinetics700-rgb.py:14]: I think it's better to reserve the `file_client_args` like [this](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/configs/_base_/datasets/coco_detection.py#L5-L14) .

**Review by hukkai — APPROVED** [review]

**Pull request merged**